src/main/resources/avro/variants.avdl

```diff
@@ -180,8 +264,10 @@ record Variant {
   /**
   The reference on which this variant occurs.
   (e.g. `chr20` or `X`)
+
+  If the API server supports the "classic" mode, this field must not be null.
   */
-  string referenceName;
```

Comment:
Shouldn't all of this `referenceName, start, end` data be encapsulated in a `Position` object that can be nulled, instead of requiring each field to be nulled?